### ISSUE TITLE
Ignoring "*" still shows Xcode autogenerated files

### ISSUE THREAD

**Issue by archmage** [SlatherOrg/slather#356]
Hello! I'm running Slather locally on an Xcode project. To make sure Slather's `ignore` option in my `.slather.yml` file was working as intended, I added the following:

```
ignore:
 - "*"
```

Surprisingly, the Slather output reports that a single file is being detected! 

![image](https://user-images.githubusercontent.com/4102334/33687362-1c149198-db2c-11e7-80aa-2cc9d1fb4bb9.png)

Here's some more info on the specific file - it's an auto-generated file that I believe Xcode is producing.

![image](https://user-images.githubusercontent.com/4102334/33687511-8928873a-db2c-11e7-8846-6453ba509c47.png)

Have I set up my config wrong, or is this a Slather bug?

**Comment by kainjow** [comment]
I'm seeing this too. It appears files that aren't properly getting ignored start with "../", files that aren't relative to the project.

**Comment by Pavan96** [comment]
@archmage Have you found any solution for this?

**Comment by archmage** [comment]
@Pavan96 Nope! My usage of Slather was fleeting - my team had a relatively low investment in the outcome of my work with it. Given that the issue hasn't received any attention from project maintainers in the years between then and now, I'm inclined to think it won't get fixed any time soon.

Why do you ask? Did you run into the issue too?
